server/sonar-server/src/main/resources/org/sonar/server/user/ws/current-example.json

```diff
@@ -15,11 +15,8 @@
   ],
   "permissions": {
     "global": [
-      "admin",
```

Comment:
else it does not test that filtering is applied. It probably deserves another test.